oci-library/oci-hol/java-management/set-up-logrotate/set-up-logrotate.md

```diff
@@ -0,0 +1,138 @@
+# Set up logrotate
```

Comment:
Thinking about the procedure and flow to set up a Management Agent for JMS (https://docs.oracle.com/en-us/iaas/jms/doc/agent-management.html) the set up for logrotate comes after the configuration of the Java Usage Tracker. Should we perhaps place this lab after Lab 4: Set Up of Management Agent so that it will be easier for the customer to follow?

Reply:
good point, lets do this as part of the revamp. for now i try to minimise impact to existing labs

Reply:
got it!